packages/malloy-render/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@malloydata/malloy": "^0.0.39",
     "lodash": "^4.17.20",
+    "moment-timezone": "^0.5.43",
```

Comment:
Could we use a library that's not been deprecated?

Reply:
Agh... Didn't notice it was deprecated will find another one.

Reply:
Done.